sql/moz-fx-data-shared-prod/telemetry_derived/newtab_visits_v1/schema.yaml

```diff
@@ -221,6 +221,30 @@ fields:
   - name: pocket_recommendation_id
     type: STRING
     mode: NULLABLE
+  - mode: NULLABLE
+    name: sponsored_pocket_dismissals
+    type: INTEGER
+  - mode: NULLABLE
+    name: organic_pocket_dismissals
+    type: INTEGER
+  - mode: NULLABLE
+    name: pocket_thumbs_up
+    type: INTEGER
+  - mode: NULLABLE
+    name: pocket_thumbs_down
+    type: INTEGER
+  - mode: NULLABLE
+    name: pocket_received_rank
+    type: INTEGER
+  - name: pocket_scheduled_corpus_item_id
+    type: STRING
+    mode: NULLABLE
+  - name: pocket_topic
+    type: STRING
+    mode: NULLABLE
+  - name: pocket_matches_selected_topic
+    type: STRING
+    mode: NULLABLE
```

Comment:
These fields aren't in the output so this is failing dry run and the columns are all null in the table.  This was caught by CI so please remember to check the CI status before merging.